### PR TITLE
test: increase coverage

### PR DESCRIPTION
**Pull request by Mesteery**

### CODE REVIEW

File: tap-snapshots/src/node-benchmark-compare.test.js.test.cjs [hunk 1]
```diff
@@ -123,6 +136,18 @@ In this case, there are 108 comparisons, you can thus expect the following amoun
   0.11 false positives, when considering a 0.1% risk acceptance (***)
 `
 
+exports[`src/node-benchmark-compare.test.js TAP insufficient > must match snapshot 1`] = `
+                                                                   confidence improvement accuracy (*) (**) (***)
+http/client-request-body.js method='write' len=32 type='asc' dur=5         NA  Infinity %           NA   NA    NA
+http/something.js method='write' len=32 type='asc' dur=5                   NA       NaN %           NA   NA    NA
```
Comment (Mesteery) [inline tap-snapshots/src/node-benchmark-compare.test.js.test.cjs:142]: Shouldn't it be `NA` instead of `Infinity`/`NaN`?

Comment (targos) [reply tap-snapshots/src/node-benchmark-compare.test.js.test.cjs:142]: It don't know. Ideally should be the same as with the R script

**Comment by targos** [comment]
Now that the coverage is 100%, we can remove the `--no-check-coverage` flag

File: tap-snapshots/src/node-benchmark-compare.test.js.test.cjs (hunk 1 above)
Comment (Mesteery) [reply tap-snapshots/src/node-benchmark-compare.test.js.test.cjs:142]: It is.
```
                                                                   confidence improvement accuracy (*) (**) (***)
http/client-request-body.js method='write' len=32 type='asc' dur=5         NA       Inf %           NA   NA    NA
http/something.js method='write' len=32 type='asc' dur=5                   NA       NaN %           NA   NA    NA
```

File: package.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
     "prettier-write": "prettier --write .",
     "test": "npm run eslint && npm run prettier && npm run test-only",
     "test-coverage": "npm run test-only -- --coverage-report=lcov",
-    "test-only": "tap --no-check-coverage",
-    "snap": "tap"
+    "test-only": "tap",
+    "snap": "tap --snapshot"
```
Comment (targos) [inline package.json]: "snap" actually implies de snapshot flag

Comment (Mesteery) [reply package.json]: ho, okay

**Pull request merged**